### DISCUSSION TITLE
Make GE Value for Elidinis' ward (f) the combined value of a Arcane sigil and the ward or at least the Arcane sigil

### DISCUSSION THREAD

**Discussion by KuraiOS** [runelite/runelite#15720] [Q&A] [answered]
title. it will make me feel better about bank value :)

**Comment by Nightfirecat** [comment, marked as answer]
Whoops, I missed the arcane sigil component of the ward, so the `(f)` and `(or)` variants had previously only been using the value of the base ward, and not adding the value of the sigil as well. I've corrected this in 3bdb814f33be0a94c0b3ae71158f1ed9db9a516e
